client/me/purchases/purchase-item/index.jsx

```diff
@@ -137,115 +208,151 @@ class PurchaseItem extends Component {
 		return null;
 	}
 
-	placeholder() {
-		return (
-			<span className="purchase-item__wrapper">
-				<div className="purchase-item__plan-icon" />
-				<div className="purchase-item__details">
-					<div className="purchase-item__title" />
-					<div className="purchase-item__purchase-type" />
-					<div className="purchase-item__purchase-date" />
-				</div>
-			</span>
-		);
-	}
+	getPurchaseType() {
+		const { purchase, site, translate, slug, showSite, isDisconnectedSite, name } = this.props;
+
+		if ( showSite && site ) {
+			return translate( '%(purchaseType)s for {{button}}%(site)s{{/button}}', {
+				args: {
+					purchaseType: purchaseType( purchase ),
+					site: site.name,
+				},
+				components: {
+					button: (
+						<button
+							className="purchase-item__link"
+							onClick={ ( event ) => {
+								event.stopPropagation();
+								event.preventDefault();
+								page( getPurchaseListUrlFor( slug ) );
+							} }
+							title={ translate( 'View subscriptions for %(siteName)s', {
+								args: {
+									siteName: site.name,
+								},
+							} ) }
+						/>
+					),
+				},
+			} );
+		}
 
-	scrollToTop() {
-		window.scrollTo( 0, 0 );
+		if ( isDisconnectedSite ) {
+			return translate( '%(purchaseType)s for %(site)s', {
+				args: {
+					purchaseType: purchaseType( purchase ),
+					site: name,
+				},
+			} );
+		}
+
+		return purchaseType( purchase );
 	}
 
-	renderIcon() {
+	getPaymentMethod() {
 		const { purchase } = this.props;
 
-		if ( ! purchase ) {
+		if ( isRenewing( purchase ) ) {
+			if ( purchase.payment.type === 'credit_card' ) {
+				return (
+					<>
+						<img
+							src={ getPaymentMethodImageURL( purchase.payment.creditCard.type ) }
+							alt={ purchase.payment.creditCard.type }
+							className="purchase-item__payment-method-card"
+						/>
+						{ purchase.payment.type === 'credit_card' && purchase.payment.creditCard.number }
```

Comment:
(minor) You don't need the `purchase.payment.type === 'credit_card'` because it was already checked above.

Reply:
Aah right! Thanks for catching that. I forgot to remove it from how I had the code written before.